src/debugConfigProvider.ts

```diff
@@ -162,7 +163,7 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
                     });
                 }
             }
-            else
+            else 
             {
                 platformPath = await targetCommand.deviceAppPath({
```

Comment:
Does this still work? Doesn't this also use `ios-deploy`?

Reply:
At the moment, getting the platform path still works with `ios-deploy`.

One can also do it with `devicectl` of course, for running processes as the `runningProcesses.executable` of `xcrun devicectl device info apps  --json-output -`
and for all apps installed via `apps.url` of `xcrun devicectl device info apps --json-output -`.